test/acceptance/tests/basic/basic_test.go

```diff
@@ -925,6 +1035,44 @@ func TestBasic(t *testing.T) {
 				consulServerTaskARN = tasks.TaskARNs[0]
 			})
 
+			var controllerTaskID string
+			if c.secure {
+				retry.RunWith(&retry.Timer{Timeout: 2 * time.Minute, Wait: 30 * time.Second}, t, func(r *retry.R) {
+					taskListOut := shell.RunCommandAndGetOutput(t, shell.Command{
+						Command: "aws",
+						Args: []string{
+							"ecs",
+							"list-tasks",
```

Comment:
non-blocking: It would be good to add some kind of helper function in `acceptance/framework/helpers` that runs the `ecs list-tasks` command and parses the output (I thought we had one, but no).